src/docs/goals/careers.md

```diff
@@ -7,34 +7,16 @@ featured: ../images/featured/goals.png
-  - Feels impactful. One sees the results first hand
-- Large business impact
-  - Moves some of an organizations most talented engineers around the company
-  - Able to influence a large number of projects in healthy ways
-    - Influencing as part of a team, not an outsider disconnected from implementation / results
-  - Avoids "powerpoint" architects. Keeps principal engineers writing code
-    - This helps keep them marketable. It's good for their career.
+## Diversity by Design
+
+Providing a fresh problem space with some familiarity in how to solve it can be refreshing for individual contributors. By providing value in many areas through their direct involvement, a shared goal in Aux Eng lies in providing technologists a meaningful breadth of influence. That influence comes from understanding a many of the diverse problems in a software ecosystem. Especially in senior technical tracks and leadership roles, that kind of exposure and influence in an organization goes a long way in building a career.
+
+A shared goal that benefits both the platform an engineer supports and the engineer itself is the ability to recognize patterns in a software ecosystem. Finding opportunities to shake up the meta understanding or expectations technologists hold about a platform or implementation allows for stronger practical innovation. Put another way, the diversity in problem and people exposure will let pattern recognition build better tools -- a benefit to both the platform and the career of the recognizer.
+
+## Practical Practice
+
+As platforms mature and careers mature, it's easy for individual contributors to get bogged down or caught up (depending on how the contributor sees it) in doing stuff that isn't code or practical work. A lot of strategy and planning can take over their schedule, and it's easy in those places to get disconnected from the problems being solved. Mentally and socially, it is exhausting to switch between high-level thinking and low-level implementations.
+
+It's our goal in AuxEng to provide a structure that leaves deliberate time for planning and implementation separately. Given that the practice requires a subject matter expert to be effective in running an engagement at all: this feed into the happiness of the contributor, and a secondary goal for the program. We want our people to feel like they're involved in the organization, solving real problems. We also want them to avoid being a "powerpoint architect" that barely has the chance to actually play with the software and systems they build.
+
+Through direct involvement, as well as the planning of that involvement, we grow engineers into seniority. We retain senior engineers more effectively with these programs by allowing them to "just code again". The knowledge sharing from our senior technical staff and the satisfaction they get from being able to practice their craft is a win-win-win for the platform, the contributor, and the team engaging with us.
```

Comment:
```suggestion
As platforms (and _careers_) mature, it's easy for individual contributors to get bogged down (or caught up) in doing stuff that isn't code or practical work. A lot of strategy and planning can take over their schedule, and it's easy to become disconnected from the technical problems being solved. Mentally and socially, it is exhausting to switch between high-level thinking and low-level implementations.

It's our goal in Aux Eng to provide a structure that leaves deliberate time for planning and implementation separately. Given that the practice requires a subject matter expert to be effective in running an engagement at all, this feed into the happiness of the contributor, and is a secondary goal for the program. We want our people to feel like they're involved in the organization, solving real problems. We also want them to avoid being a "PowerPoint architect" who barely has the chance to actually play with the software and systems they build.

Through direct involvement, as well as the planning of said involvement, we grow the seniority of our individual contributors. We retain senior engineers more effectively with these programs by allowing them to "just code again". The knowledge sharing from our senior technical staff, and the satisfaction they get from being able to practice their craft is a win-win-win for the platform, the contributor, and the team engaging with us.
```

^ I'm not sure I totally agree with the framing of senior engineers doing Aux so they can "just code" again, since a big part of the engagement is educational / pairing in nature, which could mean a lot of time watching _someone else_ at the terminal / IDE. Maybe we could tweak it to say that the contributor can get back to coding again _plus_ empower the host team to approach their own coding more efficiently, something like that.